tests/Dialect/Lattigo/Emitters/emit_lattigo.mlir

```diff
@@ -52,33 +52,33 @@ module {
   }
 
   // CHECK-LABEL: func test_basic_emitter
-  // CHECK: [[param:v.*]], [[err:.*]] := bgv.NewParametersFromLiteral
+  // CHECK: [[param:param.*]], [[err:.*]] := bgv.NewParametersFromLiteral
   // CHECK: bgv.ParametersLiteral
   // CHECK: LogN
   // CHECK: LogQ
   // CHECK: LogP
   // CHECK: PlaintextModulus
-  // CHECK: [[encoder:v.*]] := bgv.NewEncoder([[param]])
-  // CHECK: [[kgen:v.*]] := rlwe.NewKeyGenerator([[param]])
-  // CHECK: [[sk:v.*]], [[pk:v.*]] := [[kgen]].GenKeyPairNew()
-  // CHECK: [[rk:v.*]] := [[kgen]].GenRelinearizationKeyNew([[sk]])
-  // CHECK: [[gk5:v.*]] := [[kgen]].GenGaloisKeyNew(5, [[sk]])
-  // CHECK: [[evalKeySet:v.*]] := rlwe.NewMemEvaluationKeySet([[rk]], [[gk5]])
-  // CHECK: [[enc:v.*]] := rlwe.NewEncryptor([[param]], [[pk]])
-  // CHECK: [[dec:v.*]] := rlwe.NewDecryptor([[param]], [[sk]])
-  // CHECK: [[eval:v.*]] := bgv.NewEvaluator([[param]], [[evalKeySet]])
+  // CHECK: [[encoder:encoder.*]] := bgv.NewEncoder([[param]])
```

Comment:
will merge this PR as-is, but in future it may be simpler to just capture these via regex instead of depending on the suggested name, e.g., 

`[[encoder:[^, ].*]]`